### PR TITLE
README Revisions

### PR DESCRIPTION
**Pull request by bdice**
## Description
Revising README, improving quick access to key info and visual appeal.

## Types of Changes
- [x] Documentation update

This pull request is based on
- [x] *develop*, because it's meant to be released with 1.0

## Checklist:
- [x] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/signac/blob/master/ContributorAgreement.md).
- [x] My name is on the [list of contributors](https://github.com/glotzerlab/signac/blob/master/contributors.txt).
- [x] My code follows the [code style guideline](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md#code-style) of this project.
- [x] The changes introduced by this pull request are covered by existing or newly introduced tests.

### CODE REVIEW

**Comment by bdice** [comment]
> 
> 
> We could add PyPI download stats: PyPI

We should probably have just one downloads badge, since it's confusing to compare conda-forge total downloads with PyPI monthly downloads. Which do you prefer @csadorf?

**Comment by csadorf** [comment]
> > We could add PyPI download stats: PyPI
> 
> We should probably have just one downloads badge, since it's confusing to compare conda-forge total downloads with PyPI monthly downloads. Which do you prefer @csadorf?

Yes, we shouldn't have both. I think the PyPI one is more meaningful, but that's just my unqualified opinion.

**Comment by csadorf** [comment]
I've pushed an update where I added the PyPI badge, rearranged the badges order, and moved the logo. I think we should at least remove a few of the badges prior to merging.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
-# signac - simple data management
+# <img src="doc/images/logo.png" width="100" height="100"> Signac - Simple Data Management
```
Comment (bdice) [inline README.md]: @csadorf Did you mean to capitalize the name of the package? It's consistently lowercase everywhere else.

Comment (csadorf) [reply README.md]: No, it should not be capitalized. My mistake.

**Comment by csadorf** [comment]
@bdice I've changed the base of this PR to release/1.0.0.

**Pull request merged**